### PR TITLE
0.3.3 #13 make_backup option

### PR DESCRIPTION
**Pull request by mblakele**
@jlevy review?

### CODE REVIEW

File: instaclone/configs.py [hunk 1]
```diff
@@ -176,6 +177,12 @@ def _parse_and_validate(raw_config_list):
     except KeyError:
       raise ConfigError("invalid copy type: %s" % raw["install_method"])
 
+    # Parse booleans.
+    try:
+      raw["make_backup"] = raw["make_backup"]
+    except KeyError:
+      raise ConfigError("invalid copy type: %s" % raw["make_backup"])
```
Comment (jlevy) [inline instaclone/configs.py]: Typo in message.

File: instaclone/configs.py [hunk 2]
```diff
@@ -176,6 +177,12 @@ def _parse_and_validate(raw_config_list):
     except KeyError:
       raise ConfigError("invalid copy type: %s" % raw["install_method"])
```
Comment (jlevy) [inline instaclone/configs.py]: Typo in message.

File: instaclone/configs.py [hunk 3]
```diff
@@ -176,6 +177,12 @@ def _parse_and_validate(raw_config_list):
...
 
+    # Parse booleans.
+    try:
+      raw["make_backup"] = raw["make_backup"]
```
Comment (jlevy) [inline instaclone/configs.py]: Missing the bool parsing?

Comment (mblakele) [inline instaclone/configs.py]: If the string is `False` or `True` then by this point it's a bool already.

**Pull request merged**